src/components/ResetPasswordForm.jsx

```diff
@@ -42,12 +42,12 @@ const ResetPasswordForm = (props, routeProps) => {
 	};
 
 	const signIn = () =>{
-		props.setFieldValue('hasPasswordReset', true);
+		SetFieldValues(props, {hasPasswordReset: true});
```

Comment:
not blocking, but something to keep in mind, it looks like there are some of these fields PascalCase and some CamelCase. Not sure if that means something, or if its just a matter of one was typed one way, and one the other. Consistency would be ideal, so if this does have some reasoning it would be good to document it.

Reply:
These should be consistent. I'm unaware of any reason why some are one way or other. This should definitely be made into an issue for us to clean up. It makes me kind of itchy to be honest.